### PR TITLE
fix: console devtools service exposure

### PR DESCRIPTION
**Pull request by kpanot**

### CODE REVIEW

**Review by matthieu-crouzet — DISMISSED** [review]
In the doc we are missing that we can activate all this service through token injection instead of code in a component

File: docs/dev-tools/console-devtools.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+})
+export class AppComponent {
+  constructor(
+    applicationDevtoolsConsoleService: ApplicationDevtoolsConsoleService,
+    configurationConsoleService: ConfigurationDevtoolsConsoleService,
+    localizationConsoleService: LocalizationDevtoolsConsoleService,
+    rulesEngineDevtoolsConsoleService: RulesEngineDevtoolsConsoleService) {
+    this.getStaticConfig();
+    if (environment.DEBUG_MODE) {
+      // The COnsole Devtools services should be activated only in the development mode
```
Comment (fpaul-1A) [inline docs/dev-tools/console-devtools.md:57]:
```suggestion
      // The Console Devtools services should be activated only in the development mode
```

**Review by fpaul-1A — DISMISSED** [review]

File: docs/dev-tools/console-devtools.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Otter Console DevTools
+
+Otter Framework is exposition function to the browser console to help to debug and configure your application.
+All the helpers are exposed under the variable **_OTTER_DEVTOOLS_** accessible via `window._OTTER_DEVTOOLS_`.
+
+Currently the following modules are exposing helpers in the console:
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:6]:
```suggestion
Currently the following modules expose helpers in the console:
```

File: docs/dev-tools/console-devtools.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- **Configuration**: display and update the application configuration
+- **Localization**: toggle localization key visualization
+- **RulesEngine**: display rules engine status information
+- **Application**: display application global information
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:11]:
```suggestion
- **Application**: displays application global information
```

File: docs/dev-tools/console-devtools.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# Otter Console DevTools
+
+Otter Framework is exposition function to the browser console to help to debug and configure your application.
+All the helpers are exposed under the variable **_OTTER_DEVTOOLS_** accessible via `window._OTTER_DEVTOOLS_`.
+
+Currently the following modules are exposing helpers in the console:
+
+- **Configuration**: display and update the application configuration
+- **Localization**: toggle localization key visualization
+- **RulesEngine**: display rules engine status information
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:10]:
```suggestion
- **RulesEngine**: displays rules engine status information
```

File: docs/dev-tools/console-devtools.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
+# Otter Console DevTools
+
+Otter Framework is exposition function to the browser console to help to debug and configure your application.
+All the helpers are exposed under the variable **_OTTER_DEVTOOLS_** accessible via `window._OTTER_DEVTOOLS_`.
+
+Currently the following modules are exposing helpers in the console:
+
+- **Configuration**: display and update the application configuration
+- **Localization**: toggle localization key visualization
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:9]:
```suggestion
- **Localization**: toggles localization key visualization
```

File: docs/dev-tools/console-devtools.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+# Otter Console DevTools
+
+Otter Framework is exposition function to the browser console to help to debug and configure your application.
+All the helpers are exposed under the variable **_OTTER_DEVTOOLS_** accessible via `window._OTTER_DEVTOOLS_`.
+
+Currently the following modules are exposing helpers in the console:
+
+- **Configuration**: display and update the application configuration
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:8]:
```suggestion
- **Configuration**: displays and updates the application configuration
```

File: docs/dev-tools/console-devtools.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    LocalizationDevtoolsModule,
+    RulesEngineDevtoolsModule
+  ]
+})
+export class AppModule {
+}
+
+```
+
+Then the services activation can be done in the AppComponent as following:
```
Comment (cpaulve-1A) [inline docs/dev-tools/console-devtools.md:38]:
```suggestion
Then the services activation can be done in the AppComponent as follows:
```

**Review by matthieu-crouzet — APPROVED** [review]

**Review by cpaulve-1A — APPROVED** [review]

**Pull request merged**